tasks/check_einstein.py

```diff
@@ -0,0 +1,23 @@
+import json
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class CheckPermSetLicenses(BaseSalesforceApiTask):
+    task_options = {
+        "permission_sets": {
+            "description": "List of permission set names to check for, (ex: EinsteinAnalyticsUser)",
+            "required": True,
+        }
+    }
+
+    def _run_task(self):
+        query = self._get_query()
+        print(query)
```

Comment:
Use self.logger or remove this; print goes into the ether on MetaCI

Reply:
Missed that one, thanks.
```suggestion
```